src/CommandLineUtils/CommandLineApplication.cs

```diff
@@ -397,12 +397,13 @@ public int Execute(params string[] args)
         /// Helper method that adds a help option.
         /// </summary>
         /// <param name="template"></param>
+        /// <param name="inherited"></param>
         /// <returns></returns>
-        public CommandOption HelpOption(string template)
+        public CommandOption HelpOption(string template, bool inherited = false)
```

Comment:
Technically, this is a breaking change in .NET, even though the parameter is optional. Can you add an overload instead?

Reply:
yea I can do that!